src/models/Wallet.ts

```diff
@@ -23,11 +24,16 @@ export default class Wallet implements ISigner {
     this.info = await this.qjsWallet!.getInfo();
   }
 
-  public send = async (to: string, amount: number): Promise<Insight.ISendRawTxResult> => {
+  /*
+  * @param amount: (unit - whole QTUM)
+  */
```

Comment:
forgot the rest of the comments? 

oh btw,
```
/**
 *
 */
```
Seems to be the accepted format for comments in VSCode. If you type `/**` and press enter it will auto format it for you.